### ISSUE TITLE
Install Problem

### ISSUE THREAD

**Issue by sflavell** [mintchaos/django_compressor#1]
python setup.py install doesn't build correctly
The following directories /conf/ & /templates/... are not transferred to /site-packages/compressor/

The fix to setup.py

line 3

```
  from distutils.sysconfig import get_python_lib
```

line 22

```
  'compressor.conf',
```

line 27,29

```
  data_files=[
       ("%s/compressor/templates/compressor" % get_python_lib(), ['compressor/templates/compressor/css.html', 'compressor/templates/compressor/js.html']),
  ],
```

Regards Shane

**Comment by gcaprio** [comment]
So is this project active any more?

**Comment by mintchaos** [comment]
This should be fixed by 3b5fc5204c13f5aef90270fe5ca9fc63d784fa74
